docs/getting-started.md

```diff
@@ -111,7 +111,8 @@ From a PowerShell session run the following commands:
 
 ```PowerShell
 # Download and extract desired containerd Windows binaries
-$Version="1.6.4"
+# $Version must be a lastest value like todat 31 May 2023, when i tried to install containerd on WIN , it failed since https://github.com/containerd/containerd/releases/download/v1.6.4 does not exists but https://github.com/containerd/containerd/releases/download/v1.7.1 exists. So $Vesrion = "1.7.1" is valid today, look for current version if it fails .
+$Version="1.7.1"
```

Comment:
Couple things:

1. 1.6.4 exists for me, so not sure what happened when you tried. Could you make sure the variable contains the correct data? Maybe there was a mishap
2. 1.6.4 was pretty far behind the latest 1.6 (1.6.21) which is what we should be encouraging folks who want an LTS release to be on. If a build can't be found, we should make an issue and figure out why, not recommend to skip to the next minor release.
3. If someone were to read this, reading "when I tried to do X, it didn't work" isn't really what you want to read in a getting-started guide. The reader doesn't know who "I" is unless they went git perusing (which they shouldn't need to hopefully for just reading markdown), and it's not all too helpful